examples/website/geojson/app.js

```diff
@@ -1,8 +1,12 @@
+/* global setInterval */
```

Comment:
Do not change the existing examples. We will create a new example to showcase this feature. Separate PR.